src/ViewHelpers/ConstantViewHelper.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+use TYPO3Fluid\Fluid\Core\ViewHelper\Traits\CompileWithRenderStatic;
+
+/**
+ * Wrapper for PHPs :php:`constant` function.
+ * See https://www.php.net/manual/function.constant.php.
+ *
+ * Examples
+ * ========
+ *
+ * Get built-in PHP constant
+ * -------------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: 'PHP_INT_MAX')}
+ *
+ * Output::
+ *
+ *    9223372036854775807
+ *    (Depending on CPU architecture).
+ *
+ * Get class constant
+ * ------------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Class::CONSTANT')}
+ *
+ * Get enum value
+ * --------------
+ *
+ * ::
+ *
+ *    {f:constant(value: '\Vendor\Package\Enum::CASE')}
```

Comment:
We should clarify if the leading backslash is required or optional.

Reply:
Apparently both cases work, at least the tests didn't fail locally.

Reply:
Can you add a test for this? I think that one (either constant or one of the enums) should be enough to cover this.

I'm not sure, but this might get relevant if we start to mess with cache representations of Fluid templates.